package.json

```diff
@@ -106,6 +106,7 @@
     "@sanity/uuid": "^3.0.2",
     "@types/jest": "^29.5.6",
     "@types/node": "^14.18.9",
+    "@types/progress-stream": "^2.0.5",
```

Comment:
this needs to be in `packages/sanity/packages.json` that should fix the type issues